contracts/common/interfaces/IGateSealFactory.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+// See contracts/COMPILERS.md
+// solhint-disable-next-line
+pragma solidity >=0.4.24 <0.9.0;
+
+interface IGateSealFactory {
```

Comment:
shall we add a link to https://github.com/lidofinance/gate-seals/blob/main/contracts/GateSealFactory.vy?